[2024-11-21T12:11:21.771+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-21T12:11:22.027+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_world_dag.inaccurate scheduled__2024-11-20T00:00:00+00:00 [queued]>
[2024-11-21T12:11:22.060+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_world_dag.inaccurate scheduled__2024-11-20T00:00:00+00:00 [queued]>
[2024-11-21T12:11:22.061+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-21T12:11:23.520+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): inaccurate> on 2024-11-20 00:00:00+00:00
[2024-11-21T12:11:23.565+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=110) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-21T12:11:23.584+0000] {standard_task_runner.py:72} INFO - Started process 114 to run task
[2024-11-21T12:11:23.579+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'hello_world_dag', 'inaccurate', 'scheduled__2024-11-20T00:00:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/hello_world_dag.py', '--cfg-path', '/tmp/tmpvrejt_oo']
[2024-11-21T12:11:23.600+0000] {standard_task_runner.py:105} INFO - Job 57: Subtask inaccurate
[2024-11-21T12:11:24.733+0000] {task_command.py:467} INFO - Running <TaskInstance: hello_world_dag.inaccurate scheduled__2024-11-20T00:00:00+00:00 [running]> on host e14bbb8637e8
[2024-11-21T12:11:25.609+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hello_world_dag' AIRFLOW_CTX_TASK_ID='inaccurate' AIRFLOW_CTX_EXECUTION_DATE='2024-11-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-20T00:00:00+00:00'
[2024-11-21T12:11:25.610+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-21T12:11:25.610+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-21T12:11:25.610+0000] {logging_mixin.py:190} INFO - Current task name:inaccurate state:running start_date:2024-11-21 12:11:22.028476+00:00
[2024-11-21T12:11:25.611+0000] {logging_mixin.py:190} INFO - Dag name:hello_world_dag and current dag run status:running
[2024-11-21T12:11:25.611+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-21T12:11:25.612+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-11-21T12:11:25.640+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'inaccurate'"]
[2024-11-21T12:11:25.803+0000] {subprocess.py:99} INFO - Output:
[2024-11-21T12:11:25.816+0000] {subprocess.py:106} INFO - inaccurate
[2024-11-21T12:11:25.839+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-11-21T12:11:26.086+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-21T12:11:26.086+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=hello_world_dag, task_id=inaccurate, run_id=scheduled__2024-11-20T00:00:00+00:00, execution_date=20241120T000000, start_date=20241121T121122, end_date=20241121T121126
[2024-11-21T12:11:26.174+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-21T12:11:26.175+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-21T12:11:26.176+0000] {logging_mixin.py:190} INFO - Dag name:hello_world_dag queued_at:2024-11-21 12:09:50.298752+00:00
[2024-11-21T12:11:26.179+0000] {logging_mixin.py:190} INFO - Task hostname:e14bbb8637e8 operator:BashOperator
[2024-11-21T12:11:26.350+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-21T12:11:26.454+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-21T12:11:26.480+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
